src/docfx/restore/PackageResolver.cs

```diff
@@ -174,6 +177,67 @@ private void DownloadGitRepositoryCore(string cwd, string url, string committish
             }
         }
 
+        private void ThrowRestoreDependentRepositoryFailureError(string url, string committish, Exception ex)
+        {
+            var systemServiceTokenAccountType = "SystemServiceAccount";
```

Comment:
The value is passed in as an environment variable. We can add a comment if required -> value is from enum defined in ops

Reply:
how about duplicating the enum here?

Reply:
I would not prefer duplicating it here. @fenxu your thoughts?

Reply:
Let's add enum here so that no need to compare string everywhere.

---
In reply to: [446616336](https://github.com/dotnet/docfx/pull/6138#discussion_r446616336) [](ancestors = 446616336)

Reply:
ok